code/Model/RedirectorPage.php

```diff
@@ -194,33 +198,28 @@ protected function onBeforeWrite()
     public function getCMSFields()
     {
         $this->beforeUpdateCMSFields(function (FieldList $fields) {
-            $fields->removeByName('Content', true);
-
             // Remove all metadata fields, does not apply for redirector pages
             $fields->removeByName('Metadata');
```

Comment:
This is a composite field that gets manually added, so we can't use `ignoreFields` for it.